src/snowflake/connector/result_batch.py

```diff
@@ -270,18 +278,33 @@ def _download(self, **kwargs) -> "Response":
         for retry in range(MAX_DOWNLOAD_RETRY):
             try:
                 with TimerContextManager() as download_metric:
+                    chunk_url = self._remote_chunk_info.url
```

Comment:
This could be in-lined into the next expression, it doesn't seem to be reused afterwards

Reply:
I reuse it [here](https://github.com/snowflakedb/snowflake-connector-python/pull/780/files/07fef8da97cf07c8a4cc8b9500acd2ff069896c7#diff-69943088e0213b0053377355cbbb0d7264f23bde709fd06cabc763c663d536f3R301) (so that the error message contains the url)